packages/block-editor/src/components/inner-blocks/template-picker.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, IconButton, Placeholder } from '@wordpress/components';
+
+function InnerBlocksTemplatePicker( {
+	options,
+	onSelect,
+	allowSkip,
+} ) {
+	const classes = classnames( 'block-editor-inner-blocks__template-picker', {
+		'has-many-options': options.length > 4,
+	} );
+
+	const instructions = allowSkip ?
+		__( 'Select a layout to start with, or make one yourself.' ) :
+		__( 'Select a layout to start with.' );
+
+	return (
+		<Placeholder
+			icon="layout"
+			label={ __( 'Choose Layout' ) }
+			instructions={ instructions }
+			className={ classes }
+		>
+			{
+				/*
+				* Disable reason: The `list` ARIA role is redundant but
```

Comment:
Not super relevant for this pull request, but is the approach we've taken that we need `role="list"` for every `ul` ? Especially considering it as a "bug" of a particular browser + screen reader combination, I wonder if this is something we could isolate to a Babel transform, to remove consideration from the individual developer. That, or an inverse ESLint rule to enforce it be applied.

Currently, it's far too easy to overlook (obvious by the fact I'd omitted it in my original proposal).

Reply:
Yeah, I know we did this for the Block Hierarchy as well. yeah, it does seem like we should have a rule for it.